main.go

```diff
@@ -17,12 +19,94 @@ import (
 )
 
 type IoConfig struct {
-	StdIN, StdOut   bool
 	FileIn, FileOut string
 }
 
 const connStr = "postgresql://postgres:password@database:5432/postgres?sslmode=disable"
 
+//nolint:forbidigo
+func printHelp() {
+	fmt.Println(` `)
```

Comment:
I meant creating one print with preformatted help instead of calling `fmt.Println` multiple times:
```go
const help = `
Usage:  ./bin/main COMMAND [OPTIONS]

Commands:
...
`

...

fmt.Println(help)
```